src/libraries/pkg/test/testPackages.proj

```diff
@@ -140,15 +123,14 @@
       <_projectFile>%(SupportedPackage.ProjectFile)</_projectFile>
       <_packageId>%(SupportedPackage.Identity)</_packageId>
       <_packageVersion>%(SupportedPackage.Version)</_packageVersion>
-      <_projectTFM>%(SupportedPackage.TargetFrameworkShort)</_projectTFM>
-      <_projectRIDs>%(SupportedPackage.RuntimeIdentifiers)</_projectRIDs>
+      <_projectTFMs>@(SupportedPackage->'%(TargetFrameworkShort)')</_projectTFMs>
```

Comment:
Why are we using tfms here instead of tfm ?

Reply:
By eliminating RIDs I'm able to use a single project for testing.   RIDs required separate projects per TFM since they had to set different RIDs per TFM.  This improves performance of the tests and simplifies the process.

Reply:
would it make sense to just make arcade task return tfms object here ?